### ISSUE TITLE
DataDog Access Documentation

### ISSUE THREAD

**Issue by TaiWilkin** [department-of-veterans-affairs/va.gov-team#77264]
### Description

In the documentation for getting access to DataDog, `thevanotify-va-datadog` is listed as the default application to request access for. I believe that this is causing confusion, as many VFS developers primarily require `vagov-va-datadog` access, and I've seen a few help requests in Slack from people who asked for the incorrect hosting location. It would be helpful to have information in the access request documentation describing what each hosting location is used for, and direct users to identify the ones that are relevant to them, instead of listing one host as a default. 

### Relevant URLs

https://depo-platform-documentation.scrollhelp.site/developer-docs/get-access-to-datadog

### Which type of team are you on? (Platform team, VFS team, or Leadership)

VFS Team

**Comment by jknipes** [comment]
Hi @TaiWilkin , thank you for flagging this. We'll take a look, and review with some engineers to determine the best course of action for this page, and will work to get any updates reflected on the website.
We will provide further updates in this ticket. Please feel free to reach out if there are any questions.

**Comment by jknipes** [comment]
I sent a request to ECC Group to try to learn more about the hosting locations

**Comment by jknipes** [comment]
@TaiWilkin we have made some updates to that page since first acknowledging your request.
Could you please review [the page ](https://depo-platform-documentation.scrollhelp.site/developer-docs/get-access-to-datadog)again to see if what you are looking for has been updated?

**Comment by TaiWilkin** [comment]
No, I'm not seeing any information yet on which hosting locations are for which purpose. Thank you for looking into this!

**Comment by jknipes** [comment]
@va-albers would you have any information regarding the various hosting locations?

**Comment by jknipes** [comment]
@TaiWilkin this was corrected at some point in the past because there is no longer any mention of the `vanotify-va-datadog` instance. Currently on that article, it only lists the `vagov-va-datadog` instance, which is the instance majority of users will be seeking access to.
Closing this issue.
